front/app/modules/commercial/user_custom_fields/citizen/components/UserCustomFieldsSignUpInModal.tsx

```diff
@@ -65,7 +65,6 @@ const UserCustomFieldsSignUpInModal = memo<Props>(
       ? 820
       : 580;
 
-    console.log(signUpActiveStep, metaData);
```

Comment:
Nice, thanks for catching this, I think we need a lint rule for console logs as well.

Reply:
From time to time I search for console.log, but yeah indeed.